toolkit/tools/imagegen/configuration/configuration.go

```diff
@@ -130,6 +105,51 @@ type Config struct {
 	DefaultSystemConfig *SystemConfig // A system configuration with the "IsDefault" field set or the first system configuration if there is no explicit default.
 }
 
+// IsValid returns an error if the SystemConfig is not valid
```

Comment:
>SystemConfig [](start = 35, length = 12)

nit - This function checks if the full config is valid and not necessarily just SystemConfig

Reply:
Fixed, left over from a copy/paste.